packages/cli/src/commands/new/project.ts

```diff
@@ -126,54 +126,58 @@ export const parseConfig = async (
   flags: Partial<ProjectInitializerConfig>,
   boosterVersion: string
 ): Promise<ProjectInitializerConfig> => {
-  if (flags.default) {
+  if (flags.interactive) {
+    const description = await prompter.defaultOrPrompt(
+      flags.description,
+      'What\'s your project description? (default: "")'
+    )
+    const versionPrompt = await prompter.defaultOrPrompt(flags.version, "What's the first version? (default: 0.1.0)")
+    const version = versionPrompt || '0.1.0'
+    const author = await prompter.defaultOrPrompt(flags.author, 'Who\'s the author? (default: "")')
+    const homepage = await prompter.defaultOrPrompt(flags.homepage, 'What\'s the website? (default: "")')
+    const licensePrompt = await prompter.defaultOrPrompt(
+      flags.license,
+      'What license will you be publishing this under? (default: MIT)'
+    )
+    const license = licensePrompt || 'MIT'
+    const repository = await prompter.defaultOrPrompt(
+      flags.repository,
+      'What\'s the URL of the repository? (default: "")'
+    )
+    const providerPackageName = await getProviderPackageName(prompter, flags.providerPackageName)
+
+    return Promise.resolve({
+      projectName: flags.projectName as string,
+      providerPackageName,
+      description,
+      version,
+      author,
+      homepage,
+      license,
+      repository,
+      boosterVersion,
+      installDependencies: flags.installDependencies || false,
+      initializeGit: flags.initializeGit || false,
+      interactive: true,
+    }) 
+  } else {
+    const providerPackageName = await getProviderPackageName(prompter, flags.providerPackageName)
```

Comment:
I think that in non-interactive mode we should fail and show an error message requesting the user to manually set the provider using a new `--provider` flag or instructing them to use the `--interactive` mode, something like:

```sh
$ boost new:project
ERROR: You must set a provider runtime package using the `--provider` flag or use the interactive mode with the `--interactive` flag.

Examples:

    boost new:project --provider @boostercloud/framework-provider-aws
    
  or

    boost new:project --interactive # This will show an interactive prompt that will guide you through the setup of your project.
    
<Additionally, print here the full --help message>
```

Reply:
okay gotcha!

Reply:
i added a small error message. how should I improve it? is there another way to send a message other than `throw`?